rodan-main/code/rodan/jobs/MEI_encoding/build_mei_file.py

```diff
@@ -471,7 +476,18 @@ def get_custos_pitch_heuristic(all_glyphs, custos_glyph, max_distance_to_next_cl
     next_neume_glyph = all_glyphs[next_neume_index]
     return (next_neume_glyph["note"], next_neume_glyph["octave"])
 
-def build_mei(pairs: List[Tuple[List[dict], dict]], classifier: dict, width_container: dict, staves: List[dict], page: dict):
+def staff_to_columns_dict(staves: List[dict], height: int, num_columns):
+    out = {}
+    column = 0
+    for i ,staff in enumerate(staves):
+        if staff["bounding_box"]["uly"] > (column + 1)*height:
+            column += 1
+        if column == num_columns:
+            column = num_columns - 1
+        out[i] = column
+    return out
+
+def build_mei(pairs: List[Tuple[List[dict], dict]], classifier: dict, width_container: dict, staves: List[dict], page: dict, column_split_info: dict):

```

Comment:
should use an Optional[dict] here to note that column_split_info is not actually required - was confused at first, it looks like it is